pkgs/applications/misc/slweb/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv
+, fetchgit
+, redo-apenwarr
+}:
+
+stdenv.mkDerivation rec {
+  pname = "slweb";
+  version = "0.5";
+
+  src = fetchgit {
+    url = "https://git.sr.ht/~strahinja/${pname}";
+    rev = "v${version}";
+    sha256 = "sha256-GoDumiysqIkWj2HTPQv2gheYsf4fWjtCNPFS/1R0tzc=";
+  };
+
+  nativeBuildInputs = [ redo-apenwarr ];
+
+  installPhase = ''
+    runHook preInstall
+    export PREFIX=$out
+    redo install
+    runHook postInstall
+  '';
+
+  enableParallelBuilding = true;
+
+  meta = with lib; {
+    description = "A static website generator which aims at being simplistic";
+    homepage = "https://strahinja.srht.site/slweb/";
+    license = licenses.gpl3;
```

Comment:
I think this should be `licenses.gpl3Plus`

Reply:
Done